src/lib.rs

```diff
@@ -111,11 +110,9 @@ impl Piranha {
   /// Performs cleanup related to stale flags
   fn perform_cleanup(&mut self) {
     // Setup the parser for the specific language
-    let mut parser = Parser::new();
     let piranha_args = &self.piranha_arguments;
-    parser
```

Comment:
un related to this PR :| , but I observed that i was not using the `piranha_args.language()` to get the parser